PriceCalculatorKata/src/PriceCalculator/Program.cs

```diff
@@ -3,13 +3,99 @@ namespace PriceCalculator
 {
     public class Program
     {
+        private static List<IProduct> products = new List<IProduct>();
+        private static ITax tax = Tax.GetTax();
+        private static IDiscount discount =RelativeDiscount.GetDiscountInstance();
+
+        public static string DisplayMenu()
+        {
+            String message = string.Empty;
+            message += "\n\nWelcome to Price Calculator Kata \n";
+            message += "press 1 : Change Tax value \n";
+            message += "press 2 : Add new prduct \n";
+            message += "press 3 : Display product's description \n";
+            message += "press 4 : Change Relative Discount value\n";
+            message += "press # : to exit"; 
+
+            return message;
+        }
+
+        public static void PerformAction(string choice)
+        {
+            switch (choice)
+            {
+                case "1":
+                    ChangeTax();
+                    break;
+
+                case "2":
+                    GetProductInfo();
+                    break;
+
+                case "3":
+                    DisplayProducts();
+                    break;
+
+                case "4":
+                    ChangeDiscount();
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        private static void ChangeTax()
+        {
+            Console.WriteLine("Enter the new Tax");
+            string newValue = Console.ReadLine() ?? string.Empty;
+            tax.setTax(newValue);

```

Comment:
change tax here will change the price of the product. this is like defining a global variable